src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,164 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User FIRST_VALID_USER = new User("john8989", "maximus", 20);
+    private static final User SECOND_VALID_USER = new User("ann_25", "sweet_angel", 18);
+    private static final int SIZE_OF_STORAGE_WITH_VALID_USERS = 2;
+    private static final String VALID_PASSWORD = "lviv4_4";
+    private static final String VALID_LOGIN = "bob_255";
+    private static final int VALID_AGE = 25;
+    private static final String INVALID_PASSWORD = "hfgeu";
+    private static final String LOGIN_ALREADY_EXISTS = "ann_25";
+    private static final int AGE_LESS_THEN_18 = 17;
+
+    private static final int AGE_MORE_THEN_120 = 125;
+    private static final int NEGATIVE_AGE = - 5;
+    private static final String EMPTY_STRING = "";
```

Comment:
there is no need for that many constants

Reply:
you mean to use invalid data directly in test methods and not make them constants?